courses/free-courses-en.md

```diff
@@ -497,6 +497,7 @@
 ### Cuda
 
 * [CUDA Crash Course](https://www.youtube.com/playlist?list=PLxNPSjHT5qvtYRVdNN1yDcdSl39uHV_sU) - CoffeeBeforeArch
+* [CUDA Programming Course – High-Performance Computing with GPUs](https://youtu.be/86FAWCzIe_4?si=nNR5hvJKqT2-M2m9) - Elliot Arledge (FreeCodeCamp)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!

Reply:
Thank you for the feedback! Since the video is structured as a course and is over 12 hours long, I believe it aligns with the guidelines for single video courses as per the docs. Let me know if there's anything else you'd like me to adjust.

Reply:
I added the duration of the video in the file as well now for better clarity.